src/AWS.Deploy.Recipes.CDK.Common/RecipeProps.cs

```diff
@@ -139,15 +149,16 @@ public RecipeProps()
 #nullable restore warnings
 
         public RecipeProps(string stackName, string projectPath, string recipeId, string recipeVersion,
-             string? awsAccountId, string? awsRegion,  T settings)
+             string? awsAccountId, string? awsRegion,  T settings, string deploymentBundleSettings)
```

Comment:
Technically this is a breaking change for the recipes by changing the constructor signature. Since we are mostly using this for serialization it might not matter but I could see it causing problems if a user hasn't updated their CLI but did update the reference to `AWS.Deploy.Recipes.CDK.Common` to include the breaking change. The CLI would not be sending over deployment bundles which I think would try and set this non nullable string to null. To be safe we should make this setting nullable and not part of the constructor.

Reply:
Done